### PR TITLE
IS-11691: renamed 'The Microsoft Azure SQL Data Warehouse system' to …

### PR DESCRIPTION
**Pull request by tombech**
…'The Microsoft SQL Server system' and the 'The MSSQL system' to 'The Legacy Microsoft SQL system'

### CODE REVIEW

File: changelog.rst [hunk 1]
```diff
@@ -1,6 +1,10 @@
 Changelog
 =========
 
+.. _changelog_2021_17_11:
+
+*  Changed the headers for the "The Microsoft Azure SQL Data Warehouse system" to :ref:`"The Microsoft SQL Server system" <mssql-azure-dw_system>` and the "The MSSQL system" to :ref:`"The Legacy Microsoft SQL system" <mssql_system>` to reflect the fact that the former has been superceeded by the latter, which has been kept for backwards compatibility but will likely be deprecated some time in the future. Note that changing the sytem type on existing pipes can result in slight data differences due to different driver backends; the "Microsoft SQL System" uses official microsoft (ODBC) drivers while the "Legacy Microsoft SQL System" uses open source drivers. The MS ODBC drivers should support all current MS SQL Server compatible products, including Azure Synapse Analytics (previously known as Azure Datawarehouse). Note that switching from the Legacy to the preferred MS SQL System can lead to minor data differences in properties due to the different driver backends.
```
Comment (branislavjenco) [inline changelog.rst:6]: "Note that changing the sytem type on..."  typo
Azure Datawarehouse - to be precise I think it was called `Azure SQL Data Warehouse`
Duplication of "...data differences due to different driver backends" sentences

File: changelog.rst [hunk 2]
```diff
@@ -7,6 +7,10 @@ Changelog
...
 
+.. _changelog_2021_17_11:
+
+*  Changed the name of the "The Microsoft Azure SQL Data Warehouse system" to :ref:`"The Microsoft SQL Server system" <mssql-sqlserver_system>` and the "The MSSQL system" to :ref:`"The Legacy Microsoft SQL system" <mssql_system>` to reflect the fact that the former has been superceeded by the latter, which has been kept for backwards compatibility (but will likely be deprecated some time in the future). The new type ``"system:sqlserver"`` replaces the old ``"system:mssql-azure-dw"`` which is kept as an alias for now. The reccomended "Microsoft SQL System" uses official microsoft (ODBC) drivers while the "Legacy Microsoft SQL System" uses open source drivers. The MS ODBC drivers should support all current MS SQL Server compatible products, including Azure Synapse Analytics (previously known as Azure Datawarehouse). Note that switching from the Legacy (``"system:mssql``) to the preferred MS SQL System (``"system:sqlserver"`` aka ``"system:mssql-azure-dw"``) can lead to minor data differences in properties due to the different driver backends.
```
Comment (grove) [inline changelog.rst:12]: typo: reccomended

Comment (grove) [reply changelog.rst:12]: Replace "Azure Datawarehouse" with "Azure SQL Data Warehouse"? Not quite sure what the right name is, but it is inconsistent with the one in configuration.rst.

Comment (knutj42) [reply changelog.rst:12]: "the former has been superceeded by the latter" seems to have the ordering wrong?

Comment (knutj42) [reply changelog.rst]: We call the preferred system by different names in this paragraph:  "Microsoft SQL Server system", "Microsoft SQL System" and "MS SQL System". I think we should stick to one version of the name, even if the full name is a bit longer.

File: changelog.rst [hunk 3]
```diff
@@ -7,6 +7,10 @@ Changelog
...
 
+.. _changelog_2021_17_11:
+
+*  Changed the name of the "The Microsoft Azure SQL Data Warehouse system" to :ref:`"The Microsoft SQL Server system" <mssql-sqlserver_system>` and the "The MSSQL system" to :ref:`"The Legacy Microsoft SQL system" <mssql_system>` to reflect the fact that the latter has been superceeded by the former, which has been kept for backwards compatibility (but will likely be deprecated some time in the future). The new type ``"system:sqlserver"`` replaces the old ``"system:mssql-azure-dw"`` which is kept as an alias for now. The reccommended "Microsoft SQL System" uses official microsoft (ODBC) drivers while the "Legacy Microsoft SQL System" uses open source drivers. The MS ODBC drivers should support all current MS SQL Server compatible products, including Azure Synapse Analytics (previously known as Azure SQL DataWarehouse). Note that switching from the Legacy (``"system:mssql``) to the preferred MS SQL System (``"system:sqlserver"`` aka ``"system:mssql-azure-dw"``) can lead to minor data differences in properties due to the different driver backends.
```
Comment (knutj42) [inline changelog.rst:12]: "the former has been superceeded by the latter" seems to have the ordering wrong?

The preferred system is seems to be called by different names in this paragraph: "Microsoft SQL Server system", "Microsoft SQL System" and "MS SQL System". I think we should stick to one version of the name, even if the full name is a bit longer.

Comment (knutj42) [reply changelog.rst]: "that the latter has been superceeded by the former, which has been kept for backwards compatibility (but will likely be deprecated some time in the future)"  => from this sentence It is not clear to me which of the systems it is that will be deprecated. The former or the latter? Perhaps split this into two sentences to make it 100% unambiguous?

File: changelog.rst [hunk 4]
```diff
@@ -7,6 +7,10 @@ Changelog
...
 
+.. _changelog_2021_17_11:
+
+*  Changed the name of the "The Microsoft Azure SQL Data Warehouse system" to :ref:`"The Microsoft SQL Server system" <mssql-sqlserver_system>` and the "The MSSQL system" to :ref:`"The Legacy Microsoft SQL system" <mssql_system>` to reflect the fact that the latter has been superceeded by the former, which has been kept for backwards compatibility (but will likely be deprecated some time in the future). The new type ``"system:sqlserver"`` replaces the old ``"system:mssql-azure-dw"`` which is kept as an alias for now. The reccommended "Microsoft SQL Server System" uses official microsoft (ODBC) drivers while the "Legacy Microsoft SQL system" uses open source drivers. The MS ODBC drivers should support all current MS SQL Server compatible products, including Azure Synapse Analytics (previously known as Azure SQL DataWarehouse). Note that switching from the Legacy (``"system:mssql``) to the preferred Microsoft SQL Server system (``"system:sqlserver"`` aka ``"system:mssql-azure-dw"``) can lead to minor data differences in properties due to the different driver backends.
```
Comment (grove) [inline changelog.rst:12]: Typo: reccommended

Comment (grove) [reply changelog.rst:12]: Needs camel-casing: microsoft

Comment (grove) [reply changelog.rst]: Spell out "MS" as "Microsoft"?

Comment (tombech) [reply changelog.rst]: all fixed, I hope

**Review by knutj42 — APPROVED** [review]

**Pull request merged**